### ISSUE TITLE
Can not able to run build:prod

### ISSUE THREAD

**Issue by sainisagar310** [linnovate/mean#1823]
Getting error when I'm trying to build:prod => ERROR in main.4b6102611301afd1e045.bundle.js from UglifyJs
Unexpected character '`' [main.4b6102611301afd1e045.bundle.js:37478,78]

**LOG FILE**

0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Users\\Sagar.Saini\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'build:prod' ]
2 info using npm@5.3.0
3 info using node@v8.2.1
4 verbose run-script [ 'prebuild:prod', 'build:prod', 'postbuild:prod' ]
5 info lifecycle angular-starter@6.0.0~prebuild:prod: angular-starter@6.0.0
6 info lifecycle angular-starter@6.0.0~build:prod: angular-starter@6.0.0
7 verbose lifecycle angular-starter@6.0.0~build:prod: unsafe-perm in lifecycle true
8 verbose lifecycle angular-starter@6.0.0~build:prod: PATH: C:\Users\Sagar.Saini\AppData\Roaming\npm\node_modules\npm\bin\node-gyp-bin;C:\data\cloud\mean2\node_modules\.bin;C:\ProgramData\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Users\Administrator\.dnx\bin;C:\Program Files\Microsoft DNX\Dnvm\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files (x86)\Windows Kits\8.1\Windows Performance Toolkit\;C:\Program Files\TortoiseSVN\bin;C:\ProgramData\chocolatey\bin;;C:\Users\Administrator\AppData\Local\Microsoft\WindowsApps;;C:\ProgramData\chocolatey\bin;C:\Program Files\MongoDB\Server\3.2\bin;C:\Program Files\MongoDB\Server\3.2\bin;C:\Program Files (x86)\Brackets\command;C:\Program Files\nodejs\;C:\Ruby24-x64\bin;C:\Users\Sagar.Saini\AppData\Local\Microsoft\WindowsApps;C:\Users\Sagar.Saini\AppData\Local\Programs\Git\cmd;C:\Users\Sagar.Saini\AppData\Roaming\npm
9 verbose lifecycle angular-starter@6.0.0~build:prod: CWD: C:\data\cloud\mean2
10 silly lifecycle angular-starter@6.0.0~build:prod: Args: [ '/d /s /c',
10 silly lifecycle   'npm run clean:dist && npm run webpack -- --config config/webpack.prod.js  --progress --profile --bail' ]
11 silly lifecycle angular-starter@6.0.0~build:prod: Returned: code: 2  signal: null
12 info lifecycle angular-starter@6.0.0~build:prod: Failed to exec build:prod script
13 verbose stack Error: angular-starter@6.0.0 build:prod: `npm run clean:dist && npm run webpack -- --config config/webpack.prod.js  --progress --profile --bail`
13 verbose stack Exit status 2
13 verbose stack     at EventEmitter.<anonymous> (C:\Users\Sagar.Saini\AppData\Roaming\npm\node_modules\npm\lib\utils\lifecycle.js:289:16)
13 verbose stack     at emitTwo (events.js:125:13)
13 verbose stack     at EventEmitter.emit (events.js:213:7)
13 verbose stack     at ChildProcess.<anonymous> (C:\Users\Sagar.Saini\AppData\Roaming\npm\node_modules\npm\lib\utils\spawn.js:40:14)
13 verbose stack     at emitTwo (events.js:125:13)
13 verbose stack     at ChildProcess.emit (events.js:213:7)
13 verbose stack     at maybeClose (internal/child_process.js:921:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
14 verbose pkgid angular-starter@6.0.0
15 verbose cwd C:\data\cloud\mean2
16 verbose Windows_NT 10.0.14393
17 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Sagar.Saini\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "build:prod"
18 verbose node v8.2.1
19 verbose npm  v5.3.0
20 error code ELIFECYCLE
21 error errno 2
22 error angular-starter@6.0.0 build:prod: `npm run clean:dist && npm run webpack -- --config config/webpack.prod.js  --progress --profile --bail`
22 error Exit status 2
23 error Failed at the angular-starter@6.0.0 build:prod script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 2, true ]

**Comment by wizardnet972** [comment]
This happend because this file: `src\app\posts\graphql\mutations.js`

if you change ` to ' - 
for example: 

```

export const RemovePostMutation = gql(
    'mutation removePost($id: ID!) {'+
    '    removePost(id: $id) {'+
    '        id'+
    '        title'+
    '    }'+
    '}'
);
```

the aot/prod should be work...

BUT! This is temp solution. 
I think something bad going with webpack.config.prod
@liorkesos can you provide us better solution?

**Comment by timelf123** [comment]
tl;dr: uglify doesn't play well with es6
